fearless/Common/DataProvider/Sources/Rewards/SubqueryStakeSource.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+import RobinHood
+import FearlessUtils
+import BigInt
+
+final class SubqueryStakeSource {
+    typealias Model = [SubqueryStakeChangeData]
+
+    let address: String
+    let url: URL
+
+    init(
+        address: AccountAddress,
+        url: URL
+    ) {
+        self.address = address
+        self.url = url
+    }
+}
+
+extension SubqueryStakeSource: SingleValueProviderSourceProtocol {
```

Comment:
I think we should use pagination here. So consider to use DataProviderSourceProtocol